features/contract_loading.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Contract Loading
+  To interact with contracts on the platform
+  They must first be loaded
+  However loading is provably an idempotent operation
+
+  Scenario Outline: a contract cannot be called without first being loaded
+    When I subscribe by public key <pk>
+    And I call the contract <address> with the method <method> and arguments <methodArguments> knowing the contract is not initialized
+    Then <pk> should not receive a signing request
```

Comment:
Better to model this outcome as an error. While we could auto-load the contract, I'd suggest we don't as then the two operations remain independent for now